### PR TITLE
Remove base64 strict backport as it is no longer needed

### PR DESCRIPTION
**Pull request by wvanbergen**
It ships with Ruby 1.9 and up.

@ntalbott @Shopify/payments

### CODE REVIEW

File: lib/active_merchant.rb [hunk 1]
```diff
@@ -36,28 +36,13 @@
...
-end
-
+require 'base64'
 require 'securerandom'
-require 'builder'
 require 'cgi'
 require 'rexml/document'
 require 'timeout'
 require 'socket'
+require 'builder'
```
Comment (girasquid) [inline lib/active_merchant.rb]: Why move this here?

**Comment by girasquid** [comment]
:+1:

File: lib/active_merchant.rb (hunk 1 above)
Comment (wvanbergen) [inline lib/active_merchant.rb]: This is me not stashing my changes properly.

Comment (wvanbergen) [inline lib/active_merchant.rb]: Fixed.

**Comment by ntalbott** [comment]
:cool: :+1:

**Pull request merged**